canbus_receive_topic_name: /received_messages

chassis_state_topic_name: /chassis_state

node_rate: 50   # [Herz]
